### PR TITLE
Revert thread safety fix for instance_verification plugin

### PR DESCRIPTION
**Pull request by digitaltom**
Revert thread safety fix for instance_verification plugin. 

Fixes https://bugzilla.suse.com/show_bug.cgi?id=1183413

## Change Type

*Please select the correct option.*

- [x] **Bug Fix** (a non-breaking change which fixes an issue)
- [ ] **New Feature** (a non-breaking change which adds new functionality)
- [ ] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [ ] I have verified that my code follows RMT's coding standards with `rubocop`.
- [ ] I have reviewed my own code and believe that it's ready for an external review.
- [ ] I have provided comments for any hard-to-understand code.
- [ ] I have documented the `MANUAL.md` file with any changes to the user experience.
- [ ] RMT's test coverage remains at 100%.
- [ ] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

## Other Notes

Please use this space to provide notes or thoughts to the team, such as tips on how to review/demo your changes.

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless this is a trivial change, please include a CHANGELOG entry.<br />
Run <code>osc vc</code> in the <code>package</code> directory to add one.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: db/schema.rb [hunk 1]
```diff
@@ -160,6 +160,10 @@
     t.index ["login"], name: "index_systems_on_login", unique: true
   end
 
+  add_foreign_key "activations", "services", on_delete: :cascade
```
Comment (ikapelyukhin) [inline db/schema.rb:163]: Should schema changes be really part of this PR?

Comment (digitaltom) [reply db/schema.rb]: -> https://github.com/SUSE/rmt/pull/698

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/184065/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests.
Failed fast in project rmt on step docker-compose-run-tests. (Merged 053e10d3283215d6b4268b70a54d34c741a6b1a1 into 023f61747d6a2b527cbc0ac4f0a83d9845a423a3)

**Review by ikapelyukhin — APPROVED** [review]
LGTM.

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/184067/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged fdef22ebf4f899f398a6ae7a62aa2342d675e647 into 023f61747d6a2b527cbc0ac4f0a83d9845a423a3)

**Pull request merged**